### ISSUE TITLE
Finetune your 96.8 model

### ISSUE THREAD

**Issue by ntkarim** [happynear/FaceVerification#46]
Hello, thank you for all your contributions. I was planning to finetune your 96.8 model which you provided  in one of the issues (https://github.com/happynear/FaceVerification/issues/22) on my dataset. The "CASIA_train_test.prototxt" is the training.prototxt for the model you provided I think. Can you please let me know how to label the training data to finetune this? or how to generate the train.txt file?
